server/src/socobouser/schemas/SocoboUserSchema.ts

```diff
@@ -0,0 +1,34 @@
+import { Schema } from "mongoose";
+
+export const socoboUserSchema = new Schema({
+  created: {
+    required: true,
+    type: Number
+  },
+  email: {
+    type: String
+  },
+  hasTermsAccepted: {
+    type: Boolean
+  },
+  imageUrl: {
+    type: String
+  },
+  lastModified: {
+    required: true,
+    type: Number
+  },
+  password: {
+    required: true,
+    type: String
+  },
+  provider: {
+    type: String
+  },
+  role: {
+    type: String
+  },
+  username: {
+    type: String
+  }
```

Comment:
I use the option:
{
  timestamps: true
}

So I don't need to handle any created or last modified by my own. Mongoose will automatically handle these, and add the appropriate fields to the object.

Reply:
Yes, i know. But I liked my function. :-) we could change it if you like!?

Reply:
I just tried to update a user and missed to add the update type. This got me a 500. Maybe u should return a 400 Bad Request if the input json is invalid.

Reply:
No problem, this was just for information. I think it's not too bad if we use slightly different approaches, what do u think? Should we everywhere use exactly the same way of doing things?

Reply:
I think it's okay If it is not totally different.